blazor/common/data-binding/bind-entity-framework.md

```diff
@@ -410,9 +766,23 @@ To insert a new row, click the **Add** toolbar button. The new record edit form
 
 ![After Clicking a Add button in Blazor](../images/add-row.png)
 
-Clicking the **Update** toolbar button will initiate the insert action in Grid. Now, the **OnActionBegin** event will be triggered with a **RequestType** as **Save**. You can insert the record into the database (Book table) by calling the **InsertBook()** method of the **LibraryService**.
+Clicking the **Update** toolbar button will initiate the insert action in Grid. Now, the **OnActionBegin** event will be triggered with a **RequestType** as **Save**. You can insert the record into the database (Book table) by calling the **InsertBook()** method of the `ClientServices` in Blazor Web App(BlazorWebApp.Shared project) and `LibraryService` in Blazor Server App.
```

Comment:
after changing the event the sentence also need to be correct and also need to share the API link for all API's

Reply:
As per discussion this work will handle by your team.